src/archivematica/storage_service/common/helpers.py

```diff
@@ -20,7 +20,7 @@ def is_true(env_str: str) -> bool:
 
 def get_oidc_secondary_providers(
     oidc_secondary_provider_names: Iterable[str],
-) -> dict[str, dict[str, str]]:
+) -> dict[str, dict[str, object]]:
```

Comment:
```suggestion
from typing import Union
...
) -> dict[str, dict[str, Union[str, bool]]]:
```

Reply:
Ah that did the trick...thanks @replaceafill! Fixed in latest commit...